core/src/test/java/org/everit/json/schema/ArraySchemaTest.java

```diff
@@ -41,6 +41,13 @@ public void additionalItemsSchema() {
                 .build().validate(ARRAYS.get("additionalItemsSchema"));
     }
 
+    @Test
+    public void ignoreAdditionalItemsIfNoItemsSchema() {
+        ArraySchema.builder()
+            .schemaOfAdditionalItems(FalseSchema.INSTANCE)
+            .build().validate(ARRAYS.get("noItemSchema"));
```

Comment:
Hello @zgyorffi is the "noItemSchema" present in the json file? I can't see it in the commit.

Reply:
Hi @erosb!
I reused an existing one:
https://github.com/everit-org/json-schema/blob/a8dc6860bd6a918d729cceac7bc46b2c7c871cfc/core/src/test/resources/org/everit/jsonvalidator/arraytestcases.json#L2

If you prefer it I can create a separate one.

Reply:
thanks, I grepped in the wrong directory...